src/components/projects/wbs-details/project-container/project-details/project-details.tsx

```diff
@@ -78,7 +79,11 @@ const ProjectDetails: React.FC<ProjectDetailsProps> = ({ project }: ProjectDetai
   );
 
   return (
-    <PageBlock title={'Project Details'} headerRight={<b>{project.status}</b>} body={detailsBody} />
+    <PageBlock
+      title={'Project Details'}
+      headerRight={<b>{wbsStatusPipe(project.status)}</b>}
```

Comment:
<b> should be part of the pipe since the pipe is specific to this use case and this display